### ISSUE TITLE
By default don't include ./XXX alias

### ISSUE THREAD

**Issue by JoniJnm** [code-chris/lodash-loader#8]
I had to comment the line

```js
aliases["./" + n] = path.resolve(lodashDir + "/" + file);
```

Because it's causing errors in my project. I think this line is very dangerous. I can have a `./assign` file, for stuffs, and it will be changed by lodash assign.

I think also that you should add the [modularized](https://www.npmjs.com/browse/keyword/lodash-modularized) alias:

```js
aliases["lodash." + n] = path.resolve(lodashDir + "/" + file);
```

**Comment by code-chris** [comment]
Oh, yes you are right. That can be very dangerous....
Let me think about that. Then I have to prefix the aliases or something else... The modularized lodash packages are deprecated and will be removed in the upcoming major release (5.0.0). [Roadmap](https://github.com/lodash/lodash/wiki/Roadmap)

**Comment by code-chris** [comment]
Fixed with #10
